src/main/java/core/basesyntax/Lottery.java

```diff
@@ -3,10 +3,11 @@
 import java.util.Random;
 
 public class Lottery {
-    public static final int MAX_BALL_NUMBER_RANDOM = 100;
+    private static final int MAX_BALL_NUMBER_RANDOM = 100;
+    private Random randomNumberBall = new Random();
```

Comment:
Question about naming. This is just `Random` class instance, probably clarifying it as `NumberBall` is redundant.